k8s/base/app/secret.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: app-secret
+  labels:
+    app: nestjs-app
+    component: backend
+type: Opaque
+data:
+  # Base64 encoded values - CHANGE ALL VALUES FOR PRODUCTION
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)